pages/search/search.jsx

```diff
@@ -31,15 +37,21 @@ class Search extends React.Component {
   }
 
   getSearchCriteria(theProps) {
+    let featured = FEATURE_FILTER_OPTIONS.filter(option => {
+      return theProps.location.query.featured === option.value;
```

Comment:
can we rename `theProps` to just `props`? "the" as an article from natural English is already implied by virtue of this being a defined object being passed as input argument.